Mastodon/Scene/Profile/RemoteProfileViewModel.swift

```diff
@@ -46,7 +46,9 @@ final class RemoteProfileViewModel: ProfileViewModel {
                     assertionFailure()
                     return
                 }
-                self.user = mastodonUser
+                DispatchQueue.main.async {
```

Comment:
As we're currently expecting the `receiveValue` to be running on the main thread anyways, we could also add a `            .receive(on: DispatchQueue.main)` to the combine flow.